client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -105,6 +105,13 @@ class EmailPlan extends React.Component {
 			);
 	}
 
+	getAccountType() {
+		if ( this.state.emailAccounts[ 0 ] ) {
+			return this.state.emailAccounts[ 0 ].account_type ?? null;
+		}
+		return null;
```

Comment:
I wonder if we could simplify this method like this:

```javascript
return this.state?.emailAccounts[ 0 ]?.account_type ?? null;
```

Reply:
Done!